packages/xo-server-sdn-controller/src/private-network/private-network.js

```diff
@@ -18,6 +18,8 @@ export class PrivateNetwork {
     this.controller = controller
     this.uuid = uuid
     this.networks = {}
+
+    this.controller.privateNetworks[this.uuid] = this
```

Comment:
Wouldn't it make more sense to do this via a method?

Reply:
A setter? I can but I don't see the point.